apps/full-site-editing/full-site-editing-plugin/full-site-editing/plugins/close-button-override/index.js

```diff
@@ -16,9 +16,16 @@ domReady( () => {
 		return;
 	}
 
-	const closeButton = document.querySelector( '.edit-post-fullscreen-mode-close__toolbar a' );
+	const editPostHeaderInception = setInterval( () => {
```

Comment:
Is the purpose of this interval to apply our URL override after the Calypsoify/Gutenframe side has done so too?

Reply:
For dynamic elements we can probably attach a click listener to the target parent that's guaranteed to be there, and do a check on the target event for if the callback should fire. For context see examples in:

https://api.jquery.com/delegate/

Reply:
> For dynamic elements we can probably attach a click listener to the target parent that's guaranteed to be there, and do a check on the target event for if the callback should fire. 

Yes, that might be easier to follow too. IIRC I did something similar for revisions in D24735-code if you need a working example.